Cargo.toml

```diff
@@ -55,3 +55,61 @@ members = [
+libp2p-floodsub = { version = "0.43.0", path = "protocols/floodsub" }
+libp2p-gossipsub = { version = "0.45.1", path = "protocols/gossipsub" }
+libp2p-identify = { version = "0.43.0", path = "protocols/identify" }
+libp2p-identity = { version = "0.2.3" }
+libp2p-kad = { version = "0.44.4", path = "protocols/kad" }
+libp2p-mdns = { version = "0.44.0", path = "protocols/mdns" }
+libp2p-memory-connection-limits = { version = "0.1.0", path = "misc/memory-connection-limits" }
+libp2p-metrics = { version = "0.13.1", path = "misc/metrics" }
+libp2p-mplex = { version = "0.40.0", path = "muxers/mplex" }
+libp2p-muxer-test-harness = { path = "muxers/test-harness" }
+libp2p-noise = { version = "0.43.0", path = "transports/noise" }
+libp2p-perf = { version = "0.2.0", path = "protocols/perf" }
+libp2p-ping = { version = "0.43.0", path = "protocols/ping" }
+libp2p-plaintext = { version = "0.40.0", path = "transports/plaintext" }
+libp2p-pnet = { version = "0.23.0", path = "transports/pnet" }
+libp2p-quic = { version = "0.9.2", path = "transports/quic" }
+libp2p-relay = { version = "0.16.1", path = "protocols/relay" }
+libp2p-rendezvous = { version = "0.13.0", path = "protocols/rendezvous" }
+libp2p-request-response = { version = "0.25.1", path = "protocols/request-response" }
+libp2p-swarm = { version = "0.43.3", path = "swarm" }
+libp2p-swarm-derive = { version = "0.33.0", path = "swarm-derive" }
+libp2p-swarm-test = { version = "0.2.0", path = "swarm-test" }
+libp2p-tcp = { version = "0.40.0", path = "transports/tcp" }
+libp2p-tls = { version = "0.2.0", path = "transports/tls" }
+libp2p-uds = { version = "0.39.0", path = "transports/uds" }
+libp2p-wasm-ext = { version = "0.40.0", path = "transports/wasm-ext" }
+libp2p-webrtc = { version = "0.6.0-alpha", path = "transports/webrtc" }
+libp2p-websocket = { version = "0.42.0", path = "transports/websocket" }
+libp2p-webtransport-websys = { version = "0.1.0", path = "transports/webtransport-websys" }
+libp2p-yamux = { version = "0.44.1", path = "muxers/yamux" }
+multistream-select = { version = "0.13.0", path = "misc/multistream-select" }
+quick-protobuf-codec = { version = "0.2.0", path = "misc/quick-protobuf-codec" }
+quickcheck = { package = "quickcheck-ext", path = "misc/quickcheck-ext" }
+rw-stream-sink = { version = "0.4.0", path = "misc/rw-stream-sink" }
+multiaddr = "0.18.0"
+multihash = "0.19.0"
+
+
+[patch.crates-io]
+
+# Patch away `libp2p-idnentity` in our dependency tree with the workspace version.
+# `libp2p-identity` is a leaf dependency and used within `rust-multiaddr` which is **not** part of the workspace.
+# As a result, we cannot just reference the workspace version in our crates because the types would mismatch with what
+# we import via `rust-multiaddr`.
+# This is expected to stay here until we move `libp2p-identity` to a separate repository which makes the dependency relationship more obvious.
+libp2p-identity = { path = "identity" }
```

Comment:
I don't think we should introduce this in the backport. That was a feature we landed afterwards.